### PR TITLE
feat(pixie): added Helm command for redacting columns of data

### PR DESCRIPTION
**Pull request by mmfred**
Add Helm command info.

### CODE REVIEW

**Review by aimichelle — APPROVED** [review]
Looks good from a Pixie perspective. Thanks Michelle!

File: src/content/docs/kubernetes-pixie/auto-telemetry-pixie/manage-pixie-data.mdx [hunk 1]
```diff
@@ -13,7 +13,7 @@ redirects:
   - /docs/auto-telemetry-pixie/manage-pixie-data
 ---
 
-You have options for the type and the amount of data you ingest when you install Auto-telemetry with Pixie. During install, use Helm to reduce or exclude data, either by excluding specific namespaces or pods, or by collecting data for only the nodes you want.
+You have options for the type and the amount of data you ingest when you install Auto-telemetry with Pixie. During install, use Helm to reduce, restrict, or exclude data, by excluding specific namespaces or pods, by collecting data for only the nodes you want, or by redacting columns that contain sensitive information.
```
Comment (barbnewrelic) [inline src/content/docs/kubernetes-pixie/auto-telemetry-pixie/manage-pixie-data.mdx:16]: 33 word sentence is grammatically correct but would be easier to skim if you changed it to bullets.
...exclude data, by:
* excluding specific...
* collecting data only...
* redacting columns...

File: src/content/docs/kubernetes-pixie/auto-telemetry-pixie/manage-pixie-data.mdx [hunk 2]
```diff
@@ -102,4 +102,17 @@ pixie-chart:
...
+
+```
+--set pixie-chart.dataAccess=Restricted
+```
+If you don't set `dataAccess` to `Restricted`, you'll continue to get full data access, which is the default state.
+
+Currently, `Full` and `Restricted` are the only two options.
+
+* `Full`: the default option. This is implied for all clusters that have already been deployed. In this method you can freely query all tables and columns and see all data.
+* `Restricted`: When querying data, columns that may contain sensitive data, such as response/request bodies and request headers, are replaced with the `REDACTED` string. Other non-sensitive columns, such as `error code` are still shown as normal. Note that restricted data access mode is not smart. It doesn't detect whether the row actually contains sensitive data. Instead, it identifies a type of content as potentially sensitive and hides it.
```
Comment (barbnewrelic) [inline src/content/docs/kubernetes-pixie/auto-telemetry-pixie/manage-pixie-data.mdx:117]: need comma "after error code, are still"

**Review by barbnewrelic — DISMISSED** [review]
Two suggestions, take 'em or leave 'em. I don't need to review this again. After the build finishes building, make any final edits you want, then use your admin privileges to merge this PR. Thanks for adding helpful new content!

**Pull request merged**